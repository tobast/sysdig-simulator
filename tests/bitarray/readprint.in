3
11101
01011
00001
