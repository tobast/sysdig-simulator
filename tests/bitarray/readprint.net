INPUT a,b
OUTPUT z
VAR
	a:4, b, z:5
IN
z = CONCAT b a
