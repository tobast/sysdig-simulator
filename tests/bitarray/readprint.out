11110
10101
10000
